modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -4,7 +4,11 @@ The way one enables LibTorch [!cite](paszke2019pytorch) capabilities in MOOSE de
 the operating system (Linux or Mac) and if we use HPC or just a local workstation.
 
 !alert! note
-Before we review the main approaches, it is important to emphasize that
+Before trying to install MOOSE with libtorch make sure to check out the [compatibility matrix]
+(https://github.com/pytorch/pytorch/blob/main/RELEASE.md#release-compatibility-matrix)
```

Comment:
this does not seem to produce a link:

https://mooseframework.inl.gov/docs/PRs/27689/site/getting_started/installation/install_libtorch.html